sdcm/utils/ldap.py

```diff
@@ -25,6 +25,7 @@
 LDAP_PASSWORD = 'scylla'
 LDAP_ROLE = 'scylla_ldap'
 LDAP_USERS = ['scylla-qa', 'dummy-user']
+BUILDIN_USERS = ['cassandra']
```

Comment:
shouldn't we take this, if possible from the yaml file `authenticator_user`?

Reply:
> shouldn't we take this, if possible from the yaml file `authenticator_user`?

The buildin user is always 'cassandra', but authenticator_user can be assigned to other name.

Defining a BUILDIN_USERS list was suggested by @roydahan

Reply:
i actually loved the idea, but, couldn't we take it from `params.get('authenticator_user')` ?

Reply:
The buildin user is not configurable, I don't think we need to pass it by the parameter.

@roydahan what's your opinion?

Reply:
@amoskong , i don't mean to configure it from outside, but to get it from the already existing parameter

Reply:
> @amoskong , i don't mean to configure it from outside, but to get it from the already existing parameter

Fixed, thanks.